--- src/mpv-0.33.1/bootstrap.py	2021-04-05 12:45:41.000000000 -0400
+++ src/mpv-0.33.1/bootstrap.py	2021-05-08 11:21:44.000000000 -0400
@@ -5,10 +5,10 @@
 from __future__ import print_function
 import os, sys, stat, hashlib, subprocess
 
-WAFRELEASE = "waf-2.0.20"
+WAFRELEASE = "waf-2.0.9"
 WAFURLS    = ["https://waf.io/" + WAFRELEASE,
               "http://www.freehackers.org/~tnagy/release/" + WAFRELEASE]
-SHA256HASH = "bf971e98edc2414968a262c6aa6b88541a26c3cd248689c89f4c57370955ee7f"
+SHA256HASH = "2a8e0816f023995e557f79ea8940d322bec18f286917c8f9a6fa2dc3875dfa48"
 
 if os.path.exists("waf"):
     wafver = subprocess.check_output([sys.executable, './waf', '--version']).decode()
--- src/mpv-0.33.1/wscript	2021-05-08 11:09:56.000000000 -0400
+++ src/mpv-0.33.1/wscript	2021-05-08 10:36:14.000000000 -0400
@@ -242,6 +242,12 @@
         'deps': 'os-win32 || os-cygwin',
         'func': check_true
     }, {
+        'name': 'posix-spawn',
+        'desc': 'spawnp()/kill() POSIX support',
+        'func': check_statement(['spawn.h', 'signal.h'],
+            'posix_spawnp(0,0,0,0,0,0); kill(0,0)'),
+        'deps': '!mingw && !tvos',
+    }, {
         'name': 'glob-posix',
         'desc': 'glob() POSIX support',
         'deps': '!(os-win32 || os-cygwin)',
--- src/mpv-0.33.1/wscript_build.py	2021-05-08 11:10:19.000000000 -0400
+++ src/mpv-0.33.1/wscript_build.py	2021-05-08 10:35:41.000000000 -0400
@@ -209,7 +209,7 @@
     ])
 
     subprocess_c = ctx.pick_first_matching_dep([
-        ( "osdep/subprocess-posix.c",            "posix" ),
+        ( "osdep/subprocess-posix.c",            "posix-spawn" ),
         ( "osdep/subprocess-win.c",              "win32-desktop" ),
         ( "osdep/subprocess-dummy.c" ),
     ])
